### ISSUE TITLE
Advanced Web Analyzer for the Wayback Machine chrome extension

### ISSUE THREAD

**Issue by shrenikr235** [internetarchive/wayback-machine-chrome#438]
A live website analyzer that provides information including website traffic, statistics, popularity, contact info etc. would be nice to have if integrated into the wayback machine chrome extension. I'd like to work on this project if possible. I'm new here so any input is appreciated.

**Comment by shobhi1310** [comment]
yeah! this is a very nice idea, but isn't it too obvious that things like traffic, statistics, etc. are the company's personnal issues and all data is not meant for public? Just saying, even I am intersted in working towards this issue if everything is legal. 😀

**Comment by cgorringe** [comment]
I'd think much of that information would be dependent on what the website shares. The extension does have a context page for the Alexa service. Would you have any other suggestions for where we could link to such data?

**Comment by shucon** [comment]
@cgorringe Ahrefs provide a lot of data, though it's a paid service. So don't know if it's feasible.

**Comment by cgorringe** [comment]
Including this link here for future reference: https://ahrefs.com
